apps/talk.sh

```diff
@@ -20,6 +18,27 @@ debug_mode
 # Must be root
 root_check
 
+# Check if adminer ist already installed
+print_text_in_color "$ICyan" "Checking if Talk is already installed..."
+if [ -n "$(occ_command_no_check config:app:get spreed turn_servers | sed 's/\[\]//')" ]
+then
+    msg_box "It seems like 'Talk' is already installed.\nIf you continue, Talk and all Talk-settings get deleted."
+    if [[ "no" == $(ask_yes_or_no "Do you really want to continue?") ]]
+    then
+        exit
+    fi
+    print_text_in_color "$ICyan" "Uninstalling Talk and resetting all settings..."
+    occ_command_no_check config:app:delete spreed stun_servers
+    occ_command_no_check config:app:delete spreed turn_servers
+    occ_command_no_check app:remove spreed
+    rm $TURN_CONF
+    apt purge coturn -y
+    msg_box "Talk was successfully uninstalled and all settings were resetted."
+    exit
```

Comment:
Do you exit the script here? Why?

Reply:
Because I think it makes sense, if you run the script the first time and install it -> run it the second time and uninstall it -> run it the third time and reinstall it -> run it the fourth time and uninstall it again -> and so on.

Reply:
That is how I imagine it to be for all scripts, but for some scripts it is currently too complicated to implement.

Reply:
Because I think it makes sense, if you run the script the first time and install it -> run it the second time and uninstall it -> run it the third time and reinstall it -> run it the fourth time and uninstall it again -> and so on.

Reply:
That is how I imagine it to be for all appscripts, but for some scripts it is currently too complicated to implement.

Reply:
Hmm, I don't like that flow. 

Either it's installing for the first time, or it's reinstalling. For uninstall, please make a separate script, or a question, do you want to uninstall or re-install?

Reply:
Okay, then another suggestion: after the uninstallation you can choose to directly install the app again. What do you think?

Reply:
The flow should be:

1. New installation ---> install
2. Already installed ---> Warn about de-installing ----> de-install ----> ask if the user wants to reinstall---> re-install

Reply:
@szaimen When you have time, please take a look at this.

Reply:
this should be done.